src/main/java/com/ekart/config/SecurityConfig.java

```diff
@@ -35,7 +42,10 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
     @Override
     // use for authorization and form login
     protected void configure(HttpSecurity http) throws Exception {
-        http.csrf().disable();
+        // will enable this once cookie issue is resolve
+        //http.csrf().csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse());
+
+        http.cors().and().csrf().disable();
```

Comment:
why disabling the cors?

Reply:
CORS is not disabled. CSRF is disabled.

http.cors().disable() will disable cors